charts/incubator/hyperswitch-helm/templates/_helpers.tpl

```diff
@@ -31,8 +31,16 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
-{{/*Bitnami Redis host */}}
+
 {{- define "redis.svc" -}}
 {{- printf "%s-redis-master"  .Release.Name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+
+{{- define "postgresql.svc" -}}
+{{- printf "%s-postgresql" .Release.Name | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "postgresql.secret" -}}
+{{- printf "%s-postgresql" .Release.Name | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
```

Comment:
it will be better if we move `redis.svc` and `postgresql.svc` to values.yaml itself so that it will be easier to configure it someone is user external psql and redis

Reply:
good point @jagan-jaya , I added in todo list external PG and Redis configurations, I will do feature flip in next PR